src/EventuallyQueue.js

```diff
@@ -235,6 +236,7 @@ const EventuallyQueue = {
     if (queueData.length === 0) {
       return false;
     }
+    processing = true;
```

Comment:
Could `sendQueue` be called twice and lead to a race condition regarding the value `processing`?

Reply:
Processing was add for integration. testing. sendQueue should never be called twice.